x/epochs/abci_test.go

```diff
@@ -26,15 +26,15 @@ func TestEpochInfoChangesBeginEndBlockersAndInitGenesis(t *testing.T) {
 		fn                       func()
 	}{
 		{
-			expCurrentEpochStartTime: now,
+			expCurrentEpochStartTime: time.Time{},
```

Comment:
genesis.go is fixed. At currentEpoch 0, epoch start time should be in its initial, empty state.